### DISCUSSION TITLE
Render waveform just wiht the array of peaks

### DISCUSSION THREAD

**Discussion by cubancodepath** [katspaugh/wavesurfer.js#3483] [Q&A] [answered]
There is a way to use the waveform.js just having the array of peaks, cause I've to implement a player for big audios, the audio itself will be handle for an native audio element, and i have to show two waveform one will be a waveform with the wave of the audio in a reduce form, and the the other will be the waveform from and specific area del audio, selected from the small waveform. 

it will look like a minimap but doesn't work with zoom instead will be work like a region.

**Comment by katspaugh** [comment, marked as answer]
You could render two wavesurfers that share the same audio element and the same peaks, but one is zoomed in, while the other is zoomed out.

See https://wavesurfer.xyz/examples/?3fbd8d8b4b6c26435f312d5e9e09a07c

Is that what you want to achieve or did I misunderstand?

**Reply by cubancodepath** [reply]
Perefect this give the idea to start to build it, thanks for the producto that you has buit.
